tests/testthat/test-utils.R

```diff
@@ -206,11 +206,11 @@ test_that("`wb_dims()` can select content in a nice fashion with `x`", {
 
   # Selecting a row range
   dims_row1_to_5 <- "B3:L7"
-  expect_equal(wb_dims_cars(rows = 1:5), dims_row1_to_5)
+  expect_equal(wb_dims(x = mtcars, from_row = 2, from_col = "B", rows = 1:5), dims_row1_to_5)
```

Comment:
I don't like these functions that are defined on top of a testthat file and then it breaks at the bottom, how am I supposed to know what is going on? 😅

Reply:
Yeah! While writing the tests, I was seeing if it actually worked. I should have converted the tests to standalone pieces at the end.